docs/content.zh/docs/ops/metrics.md

```diff
@@ -299,15 +285,14 @@ class MyMapper extends RichMapFunction[Long, Long] {
     value
   }
 }
-
 ```
 {{< /tab >}}
 {{< /tabs >}}
 
-#### Meter
+### 仪表 Meter
 
-A `Meter` measures an average throughput. An occurrence of an event can be registered with the `markEvent()` method. Occurrence of multiple events at the same time can be registered with `markEvent(long n)` method.
-You can register a meter by calling `meter(String name, Meter meter)` on a `MetricGroup`.
+仪表 `Meter` 测量的是平均吞吐量。一个事件的发生率可以用 `markEvent()` 方法来注册。多个事件的发生率可以用 `markEvent(long n)` 方法注册。
```

Comment:
Here's a little odd.
You would consider free translation, not just literal translation.